powershell/module/module-class.ts

```diff
@@ -379,6 +379,21 @@ export class NewModuleClass extends Class {
 
   createInitAndPipeline(namespace: Namespace) {
     const $this = this;
+
+    const customEventListenerFunc = System.Func(
```

Comment:
This file change looks from the master instead of modelerfour branch

Reply:
Fixed. Thanks